src/Goodby/CSV/Import/Standard/LexerConfig.php

```diff
@@ -182,4 +187,23 @@ public function getIgnoreHeaderLine()
     {
         return $this->ignoreHeaderLine;
     }
+
+    /**
+     * Enable named columns. getIgnoreHeaderLine() must be false to use this.
+     * @param $useNamedColumns
+     * @return $this
+     */
+    public function setUseNamedColumns($useNamedColumns)
+    {
+        $this->useNamedColumns = $useNamedColumns;
+        return $this;
+    }
+
+    /**
+     * @return boolean
+     */
+    public function getUseNamedColumns()
```

Comment:
Not applicable, as this is mimicking already existing code structure. See `getIgnoreHeaderLine()` above for example.